### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by elineelorm**
- Added Tasks.java to create tasks
- Updated files to filter information with JSoup

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ListMessagesServlet.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    while (results.hasNext()) {
+      Entity entity = results.next();
+
+      String name = entity.getString("name");
+      String text = entity.getString("text");
+
+      Tasks message = new Tasks(name, text);
+      messages.add(message);
+    }
+
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/ListMessagesServlet.java]: Could you move this to a readFromDatastore method that returns a List<Tasks>. Then this method can be

Gson gson = new Gson()
response.setCOntentType(...)
response.getWriter().println(gson.toJson(readMessagesFromDatastore())

**Comment by elineelorm** [comment]
I updated the website so if you go to https://enuviadenu-sps-summer21.uc.r.appspot.com/list-messages, you will be able to see the changes.

**Review by sbahaud — APPROVED** [review]
Approved with one outstanding comment. Please fix that.

File: portfolio/src/main/java/com/google/sps/servlets/ListMessagesServlet.java (hunk 1 above)
Comment (elineelorm) [reply portfolio/src/main/java/com/google/sps/servlets/ListMessagesServlet.java]: fixed

**Pull request merged**